### ISSUE TITLE
captioning support for audio Augmentation

### ISSUE THREAD

**Issue by fwild** [WEKIT-ECS/MIRAGE-XR#901]
For accessibility reasons, we should have subtitles for all audio. There are various subtitle formats (.SRT for example), which have different features - need to investigate which ones work best for us and are widespread. This is usually done in the UI via upload of a .txt file, which also contains time codes.

**Comment by fwild** [comment]
In this context, Deogratias mentioned https://github.com/XR-Access-Initiative/chirp-captions as worth a look :)

**Comment by fwild** [comment]
In action screenshots: https://equalentry.com/accessibility-virtual-reality-captions-open-source/
